guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,27 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using LUKS during host provisioning.
+In addition, you can bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign it to your host group or select it manually during provisioning.
```

Comment:
That's a lot of "it"s. :-) 

I recommend that you write the first "it" in full so that the user knows what you are referring to.

Reply:
Reworded.